pallets/creditcoin/src/tests.rs

```diff
@@ -169,6 +169,16 @@ impl TestInfo {
 
 		(Creditcoin::deal_orders(expiration_block, deal_order_id.hash()).unwrap(), deal_order_id)
 	}
+
+	pub fn get_register_deal_msg(&self) -> Vec<u8> {
+		self.expiration_block
+			.encode()
+			.into_iter()
+			.chain(self.ask_guid.encode())
+			.chain(self.bid_guid.encode())
+			.chain(self.loan_terms.encode())
+			.collect::<Vec<u8>>()
+	}
```

Comment:
Duplicate with snippet in lib.rs. Why not leave the original `register_deal_order_message()` helper function and simply remove the `sha2_256()` call ?